static/robots.txt

```diff
@@ -0,0 +1,17 @@
+# exclude popular wikipedia snapshots published without link rel=canonical
+# context: https://github.com/ipfs/website/pull/334
+# en.wikipedia.org @ 2017-05-11
+# simple.wikipedia.org @ ?
+# ku.wikipedia.org @ 2017-05-10
+# tr.wikipedia.org @ 2017-04-30
+User-agent: *
+Disallow: /ipfs/QmXoypizjW3WknFiJnKLwHCnL72vedxjQkDDP1mXWo6uco/
```

Comment:
The trailing slash means an index.html at the root would be allowed to crawlers. I think we want to remove them. I checked with https://technicalseo.com/tools/robots-txt/